CaloReco/src/CaloHitMakerFast_module.cc

```diff
@@ -39,10 +39,11 @@ namespace mu2e {
             fhicl::Atom<art::InputTag> caloDigiCollection { Name("caloDigiCollection"), Comment("CaloDigi collection name") }; 
 	    fhicl::Atom<art::InputTag> pbttoken           { Name("ProtonBunchTimeTag"), Comment("ProtonBunchTime producer")};
             fhicl::Atom<double>        digiSampling       { Name("digiSampling"),       Comment("Digitization time sampling") }; 
-            fhicl::Atom<double>        deltaTPulses       { Name("deltaTPulses"),       Comment("Maximum time difference between two signals ") }; 
-            fhicl::Atom<double>        pulseRatioMin      { Name("pulseRatioMin"),      Comment("Min value of energy ratio between pulses ") }; 
-            fhicl::Atom<double>        pulseRatioMax      { Name("pulseRatioMax"),      Comment("Max value of energy ratio between pulses ") }; 
-            fhicl::Sequence<double>    caphriCrystalID    { Name("caphriCrystalID"),    Comment("List of caphri crystal ID") }; 
+            fhicl::Atom<double>        deltaTPulses       { Name("deltaTPulses"),       Comment("Maximum time difference between two signals") }; 
+            fhicl::Atom<double>        nPEperMeV          { Name("nPEperMeV"),          Comment("number of photo-electrons per MeV") }; 
```

Comment:
Will any of these new parameters (noiseLevel, etc) be time dependent?  If so, I suggest a long-term plan to migrate them to conditions data, though that is beyond the scope of this PR.